src/Invictus.Testing.Tests.Unit/LogicApps/AzureCloudExtensionsTests.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using Microsoft.Azure.Management.ResourceManager.Fluent;
+using Xunit;
+
+namespace Invictus.Testing.Tests.Unit.LogicApps
+{
+    public class AzureCloudExtensionsTests
```

Comment:
Maybe add some positive tests as well where we pass all combinations as inline data? (both expected and actual)

Reply:
Yeah, but then we're just copying the implementation. Not really 'testing' it, right.